src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -97,6 +97,9 @@ CHIP_ERROR GeneralCommissioningAttrAccess::Read(const ConcreteReadAttributePath
     case SupportsConcurrentConnection::Id: {
         return ReadSupportsConcurrentConnection(aEncoder);
     }
+    case Breadcrumb::Id: {
+        return aEncoder.Encode(DeviceLayer::DeviceControlServer::DeviceControlSvr().GetBreadcrumb());
```

Comment:
Breadcrumb is supposed to be non-volatile.  Why are we doing something other than just storing it in the attribute store in the normal way?

Reply:
This is just following through with the previous code that used the device control server to store the breadcrumb from the commands. I know there was slack discussion about moving this over to the attribute store, but IMO that should be a separate PR>

Reply:
That's fine, yes.  We should just make sure that work is tracked and happens....

Reply:
https://github.com/project-chip/connectedhomeip/issues/17515